pmd-dist/src/main/resources/scripts/pmd

```diff
@@ -52,7 +52,7 @@ set_lib_dir() {
     if [ -L "$0" ]; then
       local script_real_loc=$(readlink "$0")
     else
-      local script_real_loc=$0
+      local script_real_loc=${BASH_SOURCE[0]}
```

Comment:
That is a great insight.

I understand this should actually be

```suggestion
      local script_real_loc=${BASH_SOURCE[0]:-${(%):-%x}}
```

To remain compatible with zsh too.